examples/modals/basic.py

```diff
@@ -61,7 +60,7 @@ async def on_error(self, interaction: discord.Interaction, error: Exception) ->
         await interaction.response.send_message('Oops! Something went wrong.', ephemeral=True)
 
         # Make sure we know what the error actually is
-        traceback.print_exception(type(error), error, error.__traceback__)
+        logging.error('', exc_info=(type(error), error, error.__traceback__))
```

Comment:
This is pretty ugly and potentially confusing. Likewise, just an exception instance works here.
```suggestion
        logging.error('', exc_info=error)
```